Makefile

```diff
@@ -591,7 +633,9 @@ include frontends/blif/Makefile.inc
 
 OBJS += passes/hierarchy/hierarchy.o
 OBJS += passes/cmds/select.o
+ifeq ($(DISABLE_SPAWN),0)
 OBJS += passes/cmds/show.o
+endif
```

Comment:
I think this should only disable launching xdot from the show command, but should still retain the functionality of writing dot files. In some of the YosysJS demos we use `viz.js` to convert dot into svg.

Reply:
Ah that's an excellent idea actually, it is perfectly usable via e.g. `wasmtime` too. Updated.